lib/createStylelintResult.js

```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-const _ = require('lodash');
+const createPartialStylelintResult = require('./createPartialStylelintResult');
```

Comment:
I don't love this name, but this is only used internally. It should be easy enough to change later 🤷